src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```
@@ -17,3 +17,49 @@ admin_features_edit:
       id_feature: featureId
   requirements:
     featureId: \d+
+
+admin_features_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Sell\Catalog\Feature:index
+    _legacy_controller: AdminFeatures
+#    _legacy_link: AdminFeatures
+
+admin_features_search:
+  path: /
```

```suggestion
  path: /search
```
The path must not be the same as the index :thinking:

---

Just notice all other search routes are like this :/